src/plugins/intel_cpu/src/emitters/x64/jit_snippets_emitters.cpp

```diff
@@ -1246,6 +1324,16 @@ void BrgemmEmitter::kernel_execute(const brgemm_kernel_t *brg_kernel,
     (*brg_kernel)(&brgemm_p);
 }
 
+void BrgemmEmitter::print_debug_info() const {
+    std::cerr << "Emitter type name:" << get_type_name(this) << "\n";
```

Comment:
Can we also store brgemm node (or its name) in the BrgemmEmitter and print it here?

Reply:
sure, added